lib/pub_sub/poller.rb

```diff
@@ -1,31 +1,38 @@
 module PubSub
   class Poller
     def initialize(queue_url, verbose = false)
```

Comment:
You need to remove `queue_url` from here as you don't use it and in the rake task you're trying to call `new` without it.